### PR TITLE
528 Add gh issue templates and codeowners

### LINKED ISSUE #528: Add github issues templates and codeowners

### PR DESCRIPTION
**Pull request by Zejnilovic**
Adding templates for issues and codeowner for automatic reviewers. Purely organizational. Doesn't touch the code, only GitHub stuff.

This should close #528

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+## Expected behaviour
+A clear and concise description of what you expected to happen.
+
+## Screenshots
+If applicable, add screenshots to help explain your problem.
+
+## Desktop (please complete the following information):
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
```
Comment (wajda) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+---
+
+## Background
+A clear and concise description of where the limitation lies.
+
+## Feature
+A description of the requested feature.
+
+## Example [Optional]
```
Comment (wajda) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+4. See error
+
+## Expected behaviour
+A clear and concise description of what you expected to happen.
+
+## Screenshots
+If applicable, add screenshots to help explain your problem.
+
+## Desktop (please complete the following information):
+ - OS: [e.g. iOS]
```
Comment (wajda) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
```
Comment (wajda) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

Comment (wajda) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+3. Enter value '...'
+4. See error
+
+## Expected behaviour
+A clear and concise description of what you expected to happen.
+
+## Screenshots
+If applicable, add screenshots to help explain your problem.
+
+## Desktop (please complete the following information):
```
Comment (wajda) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[no-heading-punctuation] Don’t add a trailing `:` to headings](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 4 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

Comment (wajda) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

Comment (wajda) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+A simple example if applicable.
+
+## Proposed Solution [Optional]
```
Comment (wajda) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
+---
```
Comment (wajda) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Codacy Issue found: [[no-file-name-irregular-characters] Do not use `_` in a file name](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 6 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/question.md [hunk 8]
```diff
@@ -0,0 +1,13 @@
+---
+name: Question
+about: Ask a question
+labels: 'question'
+
+---
+
+## Background [Optional]
```
Comment (wajda) [inline .github/ISSUE_TEMPLATE/question.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 9]
```diff
@@ -0,0 +1,30 @@
+---
```
Comment (wajda) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[no-file-name-irregular-characters] Do not use `_` in a file name](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/question.md (hunk 8 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/question.md]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

**Comment by Zejnilovic** [comment]
Can we agree to ignore Codacy as this needs to follow GitHub standard?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+## Expected behaviour
+A clear and concise description of what you expected to happen.
+
+## Screenshots
+If applicable, add screenshots to help explain your problem.
+
+## Desktop (please complete the following information):
+- OS: [e.g. iOS]
+- Browser [e.g. chrome, safari]
+- Version [e.g. 22]
```
Comment (wajda) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 6 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/question.md (hunk 8 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/question.md]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 11]
```diff
@@ -0,0 +1,30 @@
...
+4. See error
+
+## Expected behaviour
+A clear and concise description of what you expected to happen.
+
+## Screenshots
+If applicable, add screenshots to help explain your problem.
+
+## Desktop (please complete the following information):
+- OS: [e.g. iOS]
```
Comment (wajda) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/question.md (hunk 8 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/question.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 10 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

Comment (wajda) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 6 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 9 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[no-file-name-irregular-characters] Do not use `_` in a file name](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 12]
```diff
@@ -0,0 +1,30 @@
...
+- Browser [e.g. chrome, safari]
```
Comment (wajda) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

Comment (wajda) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 5 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[no-heading-punctuation] Don’t add a trailing `:` to headings](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 11 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

Comment (wajda) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 7 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Codacy Issue found: [[no-file-name-irregular-characters] Do not use `_` in a file name](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 12 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 6 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 6 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 5 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[no-heading-punctuation] Don’t add a trailing `:` to headings](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 10 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 11 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 7 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Codacy Issue found: [[no-file-name-irregular-characters] Do not use `_` in a file name](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 11 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 10 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/question.md (hunk 8 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/question.md]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 10 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 12 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 9 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[no-file-name-irregular-characters] Do not use `_` in a file name](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 12 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 11 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 12 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

File: .github/ISSUE_TEMPLATE/question.md (hunk 8 above)
Comment (wajda) [inline .github/ISSUE_TEMPLATE/question.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4671131)

**Pull request merged**